editor/doc_data.cpp

```diff
@@ -1055,7 +1060,8 @@ Error DocData::save_classes(const String &p_default_path, const Map<String, Stri
 
 		_write_string(f, 1, "<tutorials>");
 		for (int i = 0; i < c.tutorials.size(); i++) {
-			_write_string(f, 2, "<link>" + c.tutorials.get(i).xml_escape() + "</link>");
+			String titleAttribute = (!c.tutorials.get(i).title.empty()) ? "title=\"" + c.tutorials.get(i).title.xml_escape() + "\"" : "";
```

Comment:
Use snake_case for identifiers (`title_attribute`).
The code might be a bit more readable if you assign `c.tutorials.get(i)` to a variable:
```
TutorialDoc tutorial = c.tutorials.get(i)
```

Reply:
Fixed with #3f1161b